src/Sulu/Bundle/AdminBundle/Resources/js/components/SingleSelect/tests/SingleSelect.test.js

```diff
@@ -1,7 +1,6 @@
 // @flow
-import {mount, shallow} from 'enzyme';
+import {fireEvent, render, screen} from '@testing-library/react';
```

Comment:
would be nice to use the `@testing-library/user-event` package instead of `fireEvent` 🙂 see https://github.com/sulu/sulu/pull/6783 for some examples